docs/src/pages/guides/right-to-left-v5/RtlOptOut.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { createMuiTheme, Theme } from '@material-ui/core/styles';
+import styled from '@emotion/styled';
+import { ThemeProvider } from 'emotion-theming';
+
+const Root = styled('div')((props: { theme?: Theme }) => ({
+  width: '100%',
+  marginTop: props.theme?.spacing(4),
+  marginRight: props.theme?.spacing(2),
+}));
+
+const AffectedText = styled('div')({
+  textAlign: 'right',
+});
+
+const UnaffectedText = styled('div')({
+  /* @noflip */
```

Comment:
Unfortunately this does not work. :\ Added TODO in the documentation for now in order to be investigated